reporter/src/main/kotlin/reporters/StaticHtmlReporter.kt

```diff
@@ -124,11 +178,53 @@ class StaticHtmlReporter : Reporter {
                         issueTable(reportTableModel.issueSummary)
                     }
 
+                    h2 {
+                        id = "project-dependencies"
+                        +"Projects and Dependencies"
+                    }
+
+                    details {
+                        unsafe {
+                            +"<summary>Project and Dependency Index</summary>"
+                        }
+                        ul {
+                            reportTableModel.projectDependencies.forEach { (project, projectTable) ->
+                                li {
+                                    a("#${project.id.toCoordinates()}") {
+                                        +project.id.toCoordinates()
+
+                                        if (projectTable.isExcluded()) {
```

Comment:
Would the output HTML be the same if the `if branch` was always executed, that is the `if..` was removed?

Reply:
It would not be the same. We generate  list here. All items have a common element, excluded ones are shown with the reason for exclusion.